doc/scenarioParametersConfiguration.md

```diff
@@ -44,220 +33,165 @@ parameters to display for each one. A tab is generated for each parameter group,
+| `maxValue`     | optional           | Maximum value (when applicable)                                                                                                                                                              |
+| `labels`       | optional           | a dict of labels for this parameter, the keys of the dict being the code ISO 639-1 of the language, and the values being the corresponding label                                             |
+| `options`      | optional           | a dict of additional properties to customize the behavior and rendering of the scenario parameter in the webapp. Available options will be detailed in the next sections of the current page |
 
-If you don't need changes for these behaviors, you don't nedd to edit the configuration file and can use its default
-content:
+### Parameters groups
 
-```
-const PARAMETERS = {};
-const PARAMETERS_GROUPS = {};
-const RUN_TEMPLATES = {};
-
-export const SCENARIO_PARAMETERS_CONFIG = {
-  parameters: PARAMETERS,
-  parametersGroups: PARAMETERS_GROUPS,
-  runTemplates: RUN_TEMPLATES
-};
-```
-
-_Important: You can use this file to reorganize the scenario parameters tabs by changing the parameters belonging to parameters groups BUT **the set of parameters associated to a run template must remain consistent with the solution configuration file** (Solution.yaml)._
+Parameters groups can define the following properties:
 
-### Translation labels
+| key                                          | mandatory/optional | description                                                                                                                                              |
+| -------------------------------------------- | ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                                         | mandatory          | unique id of the parameter group                                                                                                                         |
+| `parameters`                                 | mandatory          | fields to list the ids of the parameters that are part of this group                                                                                     |
+| `labels`                                     | optional           | a dict of labels for this parameter group, the keys of the dict being the code ISO 639-1 of the language,                                                |
+| and the values being the corresponding label |
+| `authorizedRoles`                            | optional           | an array of [APP_ROLES](../src/config/Profiles.js) allowed to see the scenario parameters tab (by default, all tabs are visible)                         |
```

Comment:
_Profiles.js_ doesn't exist anymore (and is it possible to define roles in web app ?) :thinking:

Reply:
Fixed the link to point to the back-end documentation with the up-to-date list of roles :pray: 
These roles can still be used in `authorizedRoles` in the parameters groups configuration, but we cannot define other roles than the existing ones.

Reply:
thank you ! :pray: